_people/sophia_charalambous.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Sophia Charalambous
+submitted: false
+gender: Female
+graduated: 2010
+careers:
+  - Journalist
+  - Reviewer
+links:
+  - type: Twitter
+    username: SophiaRubieC
+---
+
```

Comment:
I was suggesting a slightly modified version of that bio could be included as the bio for this page, rather than just linking to it.